internal/tools/wallet.go

```diff
@@ -63,21 +63,11 @@ func wallet(
 
 	key := service.Key().ToConfig()
 
-	conf := devWallet.Config{
-		Address:               fmt.Sprintf("0x%s", service.Address().String()),
-		PrivateKey:            strings.TrimPrefix(key.PrivateKey.String(), "0x"),
-		PublicKey:             strings.TrimPrefix(key.PrivateKey.PublicKey().String(), "0x"),
-		AccessNode:            walletFlags.Host,
-		AccountKeyID:          "0",
-		BaseURL:               "http://localhost:8701",
-		ContractFungibleToken: "0xee82856bf20e2aa6",
-		ContractFlowToken:     "0x0ae53cb6e3f42a79",
-		ContractFUSD:          "0xf8d6e0586b0a20c7",
-		ContractFCLCrypto:     "0xf8d6e0586b0a20c7",
-		AvatarURL:             "https://avatars.onflow.org/avatar/",
-		FlowInitAccountsNo:    "0",
-		TokenAmountFLOW:       "100.0",
-		TokenAmountFUSD:       "100.0",
```

Comment:
Are these config variables no longer needed?

Reply:
They are loaded dynamically by the fcl-dev-wallet

Reply:
Hmm, loaded dynamically from where? Does the dev wallet define defaults for these values?

I ask only because we previously had issues with missing config variables and these lines (I believe) were added as a fix. So want to make sure this doesn't cause a regression.

Reply:
@psiemens I made a PR some time back addressing this configurations, the problem was that everytime fcl team added a new required variable things broke since they had to add that config variable in multiple places (js app, go module in dev-wallet, cli), so I made it so there's only one place acting as a source to variables that are not CLI specific. The process is described here https://github.com/onflow/fcl-dev-wallet/tree/main/go